docs/ledger_signer.md

```diff
@@ -123,7 +123,7 @@ Tezos.contract
 </TabItem>
   <TabItem value="walletAPI">
 
-```js
+```js live noInline wallet
```

Comment:
The wallet props refer to BeaconWallet, but in the current example, we want to demonstrate the ledger signer. It might be more appropriate to put back ```js for this example.

Reply:
restored.